Apps/Src/Temperature.c

```diff
@@ -51,6 +62,9 @@ static inline int32_t median(int32_t a, int32_t b, int32_t c) {
  * @param boards LTC6811 data structure that contains the values of each register
  */
 void Temperature_Init(cell_asic *boards){
+// simulator bypasses ltc driver
+#ifndef SIMULATION
```

Comment:
I'm not a fan of this implementation. If we ever want to use the LTC anywhere we have to surround it with these guards. I also believe that we could test more (for example, someone trying to access the LTC without mutex protection) if we just rewrote the LTC drivers for simulation (could literally just be return some values from the json file).

Reply:
Or you could do something where if simulation is defined, replace all the LTC functions with some simulator functions. Though that might get more complicated.

Reply:
Yeah, we could simulate the LTC through the driver, but it would be a lot of work. The reason I did it this way here is because I think it's better to get something that works and doesn't simulate the LTC driver than delay getting a running simulator by a month or two to get the LTC driver properly simulated.

If you want, we can always go back later and upgrade the simulator by making a simulator of the LTC driver.

One concern I have with this though is that if we find bugs with how we are interfacing with the hardware (like the timing issues we ran into for temperature), the changes might break the simulator. So swapping in the fake values here will probably reduce simulator maintenance.

Reply:
What I imagined for an LTC simulator was pretty much having it copy data that we give it from the JSON file to the Minions Struct. I'm not sure if that is a full blow simulator or not, but if it isn't worth it then this is fine for now too.

Reply:
We could do that, but I think it would be annoying to figure out the right stuff to populate the JSON with